config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'cart_items/create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.